collector/README.md

```diff
@@ -146,9 +146,9 @@ second that contains a branch of your changes. To compare the two versions, do
 something like this:
 
 ```
-./target/release/collector bench_local $RUST_ORIGINAL Original
+./target/release/collector bench_local --id Original $RUST_ORIGINAL
 
-./target/release/collector bench_local $RUST_MODIFIED Modified
+./target/release/collector bench_local --id Modified $RUST_MODIFIED
```

Comment:
This should probably be using the diff syntax, where --rustc2 specifies the second rustc.

cc @nnethercote

Reply:
The diff syntax only works with the `profile_local` subcommand, not `bench_local`.

Reply:
Ah, right, I missed that this is the bench syntax. Seems good then.

Reply:
@mati865: thanks for fixing this, it was my oversight. I see there is another `bench_local` example on lines 198-199. Can you update those ones too? Thanks.

Reply:
Now that's my oversight, I had scrolled through the file but upon seeing words `Benchmarking on Windows` I scrolled even faster 😉 
It's merged already so I'll open new PR.